core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -3579,7 +3579,7 @@ class DAGSchedulerSuite extends SparkFunSuite with TempLocalSparkContext with Ti
     submit(reduceRdd, (0 until parts).toArray)
     completeShuffleMapStageSuccessfully(0, 0, parts)
```

Comment:
Once we reintroduce the `shuffleMergeEnabled` method, all changes to this file (DAGSchedulerSuite) related to mergerLocs.isEmpty == !shuffleMergeEnabled (and reverse) can be reverted (for the latest [diff](https://github.com/apache/spark/pull/34122/commits/3b52bed7ea4bbdb127a4711201d8c8268ab06650#))